pages/about/index.md

```diff
@@ -28,10 +49,69 @@ Would you like to get on the list and support us financially? Check out the [spo
+- **Free**. It’s a free software project: it seeks the user's liberty by offering a tool that can be used and enhanced without restrictions.
+- **Open**. As it's an open-source project and not a company or a product, it works in a change of paradigm: it does not sell anything, it moves away from fierce competition and bets for cooperation.
+- **Multicultural and inclusive**. It’s an initiative with an international and multicultural vision. It’s organic, always in constant mutation and change. It also actively demands compliance with human rights, respect for racial, sexual, and cultural diversity equality. And, of course, the equality between women and men.
+- **Transparent**. The initiative it’s built on transparency, being one of the main values of AsyncAPI, both internally and externally including its financing. All the relevant data of the project is public. The tasks that are being and will be worked on are public and participatory through Slack and GitHub.
+- **Horizontal and participative**. Its structure is based on horizontality; most of the decisions are defined as a group, including the Community. Anyone can submit improvements, lines of action, or elements to be taken into account to strengthen AsyncAPI
+- **User-friendly**. AsyncAPI bets for a seamless API development experience. Users should be able to create their first API within minutes without prior knowledge of AsyncAPI. It has to be natural, simple, and intuitive.
+
+## Funding
+
+AsyncAPI is an open source project and is therefore funded by talent and voluntary contributions. It relies on the expertise and dedication of the **Community developers** who believe in the project and contribute to its evolution.
+
+Financially, there are individual donations and contributions made by users of the project, and others that come from different companies and initiatives that trust and support AsyncAPI through sponsorship. The main sponsors that contribute to sustain this initiative are Postman, IQVIA Technology, Mulesoft, Salesforce, SAP, SLACK, Solace, and Tibco.
+
+All the information about the project's economy, the amount of the donations, the identity of the donors, the sponsors supporting the project and the use of this money is public. 99% of the funding goes through the Open Collective. [More information here](https://opencollective.com/asyncapi)
+
+Open Collective is an online crowdfunding platform for open and transparent communities. It provides the necessary tools to collect and share the finances raised with complete transparency. It is an ideal collective for communities such as collaborative groups or open source projects among others. This platform can be used to raise and transparently disburse money for the benefit of their members, their registered projects, and those who want to support them.
```

Comment:
imho this section can be completely removed as it explains what Open Collective is but the is a section about AsyncAPI